_posts/2023-11-30-indico-3-3-news.md

```diff
@@ -0,0 +1,96 @@
+---
+layout:     post
+title:      "Indico 3.3"
+date:       2023-11-30 12:00
+author:     "Indico Team"
+categories: indico release
+---
+
+Today, we are introducing Indico 3.3! This upcoming release brings a lot of new features and improvements, including a **new Check-in app**, **accessibility improvements (a11y)**, **receipt generation** and a whole lot more.
+
+> note ""
+> The full change log can be found [here](https://docs.getindico.io/en/stable/changelog/#v3-3).
+
+---
+
+## ♿️ Accessibility improvements (a11y)
+
+The accessibility of Indico has been improved in this release, in-line with the [WCAG 2.1](https://www.w3.org/TR/WCAG21/). This means that Indico is now easier to use by people with disabilities, including those who are blind, deaf, or have limited mobility. Notable improvements include:
+
+- Making language selection, timezone selection and search functionalities more accessible to those using screen readers
+- Preventing icons from being read out by screen readers
+- Including the selected language in the page metadata (for screen readers)
+
+> contribution "Community Contribution"
+> Feature kindly contributed by the **United Nations Office at Geneva**
+
+![](/assets/2023-11-30-indico-3-3-news/timezone-picker.png)
+
+## 📱 Check-in app
+
+The existing **Indico check-in mobile application** has been **completely rewritten**. Instead of a native app released on two separate platforms (Google Play Store and Apple App Store), it is now a progressive web app (PWA) which can be easily installed on the two major phone OSs (Android and iOS) and used offline. The new app is available for all Indico instances running version 3.3 - you can [try it out here](https://checkin.getindico.io/).
+
+Despite being hosted on [getindico.io](getindico.io), we do not collect any data from the app. The app only communicates with the Indico instance that you configure and all data is stored locally on the phone.
+
+<!-- Read our full in-depth blog post about the new Check-in app [here](https://getindico.io/blog/future-link-to-a-post-about-the-check-in-app/). -->
+
+<div id="checkin-app-images" class="image-container">
+  <img src="/assets/2023-11-30-indico-3-3-news/app1.png" />
+  <img src="/assets/2023-11-30-indico-3-3-news/app2.png" />
+  <img src="/assets/2023-11-30-indico-3-3-news/app3.png" />
+</div>
+
+## 🧾 Receipts
+
+A long await feature: Indico now supports the generation of fully customizable PDF documents. You can create **receipts**, **certificates of attendance** or any other document you might need. Once generated, participants can download their documents from the registration page.
```

Comment:
s/await/awaited/

Reply:
It should be 'long-awaited' with a dash.